editor/progress_dialog.cpp

```diff
@@ -155,9 +155,10 @@ void ProgressDialog::_popup() {
 
 	Ref<StyleBox> style = get_stylebox("panel", "PopupMenu");
 	ms += style->get_minimum_size();
-	for (int i = 0; i < 4; i++) {
-		main->set_margin(Margin(i), style->get_margin(Margin(i)));
-	}
+	main->set_margin(MARGIN_LEFT, style->get_margin(MARGIN_LEFT));
+	main->set_margin(MARGIN_RIGHT, -style->get_margin(MARGIN_RIGHT));
```

Comment:
Well, that's how the new anchoring system works. Now the margin is defined as the signed distance between the node side and the anchors. So here, as the right anchor is set at the end, you need a negative value for the right side of the node to be at the left of its anchor. Now controls' margins act more like "offsets" I'd say. (See our discussion with @toger5 in #9889)

Actually this makes the whole anchoring system more consistent, but this is more obvious when setting those values from the editor.

The fact is now that styleboxes and control nodes haven't the same conventions for margins anymore, while they are called the same. Maybe "margin" should be renamed to "offset".